collector-bitmovin-player/build.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
 		api "com.bitmovin.analytics:collector:${rootProject.ext.version}"
 	}
 	implementation "com.bitmovin.player:playercore:${rootProject.ext.playercoreVersion}"
+
+	testImplementation "io.mockk:mockk:1.10.6"
```

Comment:
do we really need yet another mock library? what is this for? don't we already have mockito?

Reply:
There were a lot of things that didn’t work too well with mockito and Kotlin. And with this library there is good Generic support and mocking/verifying works pretty well and cleanly. Mostly, being able to use the mockKConstructor makes a big difference - our collector wasn’t build with having tests in mind, so it’s really hard to pass any mocks in a lot of places.

Reply:
I support decision to use that library.

In other projects we are using `mockito-kotlin` but that is just wrapper arround `mockito` to solve compilation issues (you can still have runtime mocking issues). I checked some discussions on web and even author of `mockito-kotlin` would recoment to use that library:
https://www.reddit.com/r/androiddev/comments/9us0a0/mockk_vs_mockitokotlin/

@rolandgriesser I would set this dep to all modules that have unit tests, and share version number in root `build.gradle` file , so we have modules in sync (as in past when I first time touched project, I had an issue with different lib versions in different module in this project)